python/hail/genetics/pedigree.py

```diff
@@ -6,40 +6,40 @@
 class Trio(HistoryMixin):
     """Class containing information about nuclear family relatedness and sex.
 
-    :param str proband: Sample ID of proband.
+    :param str s: Sample ID of proband.
 
-    :param fam: Family ID.
-    :type fam: str or None
+    :param fam_id: Family ID.
+    :type fam_id: str or None
 
-    :param father: Sample ID of father.
-    :type father: str or None
+    :param pat_id: Sample ID of father.
+    :type pat_id: str or None
 
-    :param mother: Sample ID of mother.
-    :type mother: str or None
+    :param mat_id: Sample ID of mother.
+    :type mat_id: str or None
 
     :param is_female: Sex of proband.
     :type is_female: bool or None
     """
 
     @handle_py4j
     @record_init
-    @typecheck_method(proband=strlike,
-                      fam=nullable(strlike),
-                      father=nullable(strlike),
-                      mother=nullable(strlike),
+    @typecheck_method(s=strlike,
```

Comment:
I think this is wrong -- we should rename `s` to `pro_id` or something for consistency